snapcraft/tests/test_project_loader.py

```diff
@@ -1875,8 +1875,8 @@ def test_invalid_part_name_plugin_raises_exception(self):
             project_loader.Validator(self.data).validate)
 
         expected_message = ("The 'parts' property does not match the "
-                            "required schema: Additional properties are not "
-                            "allowed ('plugins' was unexpected)")
+                            "required schema: 'plugins' does not match "
+                            "'^(?!plugins$)[a-z0-9][a-z0-9+-\\/]*$'")
```

Comment:
can you return something here so the construct is rather something like "'plugins' is not a valid part name"?